include/dt-bindings/interrupt-controller/intel-ioapic.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef __DT_BINDING_IRQ_H
+#define __DT_BINDING_IRQ_H
+
+#define	LEVEL		0x00008000
```

Comment:
We need to prefix these defines with IRQ_TYPE_:

LEVEL -> IRQ_TYPE_LEVEL
EDGE -> IRQ_TYPE_EDGE
...